### PR TITLE
Random Choice Picker

### PR DESCRIPTION
**Pull request by VaishnaviMandloi**
### 👨‍💻 Changes proposed
Random choice picker project by using html , CSS and JavaScript , In which this help to select one choice for us out of all the given choices by us .

changes are made in folder -
 - public , file : RandomChoicePicker.html
 - assets , file : RandomChoicePicker.css
 - assets , file : RandomChoicePicker.js
 - readme is also update accordingly

### ✅ Check List (Check all the applicable boxes) 

- [x] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [x] This PR does not contain plagiarized content.
- [x] My Addition/Changes works properly and matches the overall repo pattern.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

- I'm a GSSOC'22 contributor 

- Random choice picker project by using html , CSS and JavaScript , In which this help to select one choice for us out of all the given choices by us .
- Like if someone is confused that they should go to picnic on which day she type Monday ,Tuesday , Wednesday the random choice select one of the choice for example it shows Monday. The choice are spiraled by (comma) " , " .
### 📷 Screenshots

#671 zeoOctave-js-project
#671 zeroOctave-js-project part2

### CODE REVIEW

**Comment by VaishnaviMandloi** [comment]
@Astrodevil please check my pr

**Comment by NOiR-07** [comment]
don't remove anything from readme just add your line, resolve this issue

**Comment by VaishnaviMandloi** [comment]
@NOiR-07 changes asked by you are done , 
actually I did not deleted anything from readme it was showing deletion because I added without using space
please review now

**Comment by NOiR-07** [comment]
> 

okay just update your forked repo once too

**Comment by VaishnaviMandloi** [comment]
> > 
> 
> okay just update your forked repo once too

@NOiR-07  changes are made as told by you , please review now

File: README.md [hunk 1]
```diff
@@ -251,8 +251,8 @@ git push -u origin <your_branch_name>
...
 | 145. | [Connect Four](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/connectfour.html)| 146. | [Maze Game](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/mazegame.html)| 147. | [Coin Toss Game](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/Coin-toss-game.html)
-| 148. |  [Movies Recommendation Page](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/movies_rec.html)| 149. | [Random Choice Picker](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/RandomChoicePicker.html)
-
+| 148. |  [Movies Recommendation Page](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/movies_rec.html)| 149. |  [Myntra Clone](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/myntraclone.html)|
+ 150. | [Random Choice Picker](https://zerooctave.github.io/ZeroOctave-Javascript-Projects/Public/RandomChoicePicker.html)
```
Comment (Astrodevil) [inline README.md:255]: you have done wrong addition, please put 150 in same line

Comment (VaishnaviMandloi) [reply README.md]: > you have done wrong addition, please put 150 in same line

@Astrodevil changes are done as told by you, please review now

**Pull request merged**